src/applications/vaos/new-appointment/redux/reducer.js

```diff
@@ -420,12 +473,12 @@ export default function formReducer(state = initialState, action) {
     case FORM_PAGE_FACILITY_SORT_METHOD_UPDATED: {
```

Comment:
In the ticket there's an AC for
> Track the user switching between toggles By recent location, By home address or By location.

I didn't see any new functionality for this here so I'm curious if you addressed this somewhere else? Or if it's already implemented somewhere?